### ISSUE TITLE
hi

### ISSUE THREAD

**Issue by jwijffels** [vriezer/sentiment#1]
stumbled upon your code while looking for some udpipe code which I've put online but couldn't find any more.
why don't you use R package https://cran.r-project.org/web/packages/word2vec/index.html http://bnosac.be/index.php/blog/100-word2vec-in-r instead of rebuilding glove?

**Comment by vriezer** [comment]
That's quite a coincidence, I've only made this repository public half an hour ago! I've tried using the word2vec package, but it gave me troubles when trying to estimate a model. I don't remember anymore what those troubles exactly were, but from what I remember it had something to do with my specific environment. Because I just wanted to continue with my paper, I decided to use the plain GloVe builds instead. Since then, I haven't changed any of the code. But you are right of course that the code provided here could be updated to work with the word2vec package. If you are interested, you are very welcome to open a pull request. I am currently quite busy, but I will at least put a note in the readme, mentioning the possibility of using your word2vec package as an alternative.
